layouts/notes/list.html

```
@@ -1,52 +1,53 @@
+    {{ range $paginator.Pages }}
+    <article class="mb-10 pb-8" style="border-bottom: 1px solid var(--color-border-rule);">
+        <header>
+            <span class="text-text-faint text-sm">
                 <time datetime='{{ .Date.Format "2006-01-02" }}' aria-label="Date: {{ .Date.Format (default "2006-01-02" .Site.Params.dateFormat) }}">
                     {{ .Date.Format "Jan 02, 2006" }}
                 </time>
-                 <a href="{{ .RelPermalink }}" title="Permalink to this note">#</a>
-                 <button class="tinylytics_kudos" data-path="{{ .RelPermalink }}"></button>
-            </header>
+                <a href="{{ .RelPermalink }}" title="Permalink to this note" class="text-text-muted no-underline hover:text-text-faint">#</a>
+                <button class="tinylytics_kudos" data-path="{{ .RelPermalink }}"></button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)